integration/deb/install_deb.go

```diff
@@ -0,0 +1,33 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os/exec"
+	"strconv"
+	"time"
+)
+
+const (
+	numberRetry = 5
+	waitTime    = time.Minute
+)
+
+// Try to install amazon cloud watch agent deb. If you can't install wait 1 minute
+func main() {
```

Comment:
Just a note, if you find yourself adding "retry" logic to other commands then I suggest turning this into a function like:
```
func ExecuteWithRetry(command []string, retryCount int, retryDelay time.Duration) error
```

Reply:
Going to add this.